src/txmempool.cpp

```diff
@@ -1217,30 +1217,31 @@ uint64_t CTxMemPool::CalculateDescendantMaximum(txiter entry) const {
     return maximum;
 }
 
-void CTxMemPool::CalculateAncestorData(const CTxMemPoolEntry& entry, size_t& ancestor_count, size_t& ancestor_size, CAmount& ancestor_fees) const
+std::tuple<size_t, size_t, CAmount> CTxMemPool::CalculateAncestorData(const CTxMemPoolEntry& entry) const
```

Comment:
In commit " [squashme] Use outputs and structured bindings for CalculateAncestorD…"

Squash the squashme?

Reply:
Done.